packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -224,6 +231,13 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     private afterChartDraw(chart: Chart) {
         const ctx = chart.ctx;
         if (ctx) {
+            if (this.isMouseMove) {
+                const startPixel = this.positionXMove;
+                const endPixel = this.positionXMove;
+                ctx.strokeStyle = '#9f9f9f';
+                ctx.fillStyle = '#9f9f9f';
+                this.drawSelection(chart, startPixel, endPixel);
+            }
```

Comment:
as mentioned above, this can be removed.

Reply:
Done.